src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/Expressions/Parsers/ExpressionParserTests.cs

```diff
@@ -329,7 +329,7 @@ private Expression SetupExpression(SearchParameterInfo searchParameter, string v
         {
             Expression expectedExpression = Substitute.For<Expression>();
 
-            _searchParameterExpressionParser.Parse(searchParameter, null, value).Returns(expectedExpression);
+            _searchParameterExpressionParser.Parse(searchParameter, null, null, value).Returns(expectedExpression);
```

Comment:
Any chance E2E test can be added?
Either to [https://github.com/microsoft/fhir-server/blob/main/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs](https://github.com/microsoft/fhir-server/blob/main/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs)
or [https://github.com/microsoft/fhir-server/blob/main/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/ReferenceSearchTests.cs](https://github.com/microsoft/fhir-server/blob/main/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/ReferenceSearchTests.cs)
(later sound more appropriate, but I can be wrong)

Reply:
Added in e041735